source/usage-examples/changeStream.txt

```diff
@@ -7,76 +7,80 @@ Watch for Changes
 Open a Change Stream
 --------------------
 
-You can keep track of changes to data in MongoDB, such as changes to a
-collection, database, or deployment, by opening a **change stream**. A change
-stream allows applications to watch for changes to data and react to them.
-You can open a change stream by calling the ``watch()`` method on
-a ``Collection``, ``Db``, or ``MongoClient`` object. The change stream
-returns **change event** documents when they occur.
+You can watch for changes to a single collection, a database, or an entire deployment in MongoDB with
+:manual:`Change Streams </changeStreams/>`.
+Open a change stream by calling the ``watch()`` method on a ``Collection``, ``Db``, or ``MongoClient`` object. The
+change stream emits :manual:`change event </reference/change-events/>` documents when they occur.
 
-The ``watch()`` method optionally takes an **aggregation pipeline**  which
-consists of an array of **stages** as the first parameter to filter and
-transform the change events output as follows:. 
+The ``watch()`` method optionally takes an :manual:`aggregation pipeline </reference/operator/aggregation-pipeline/>`
+which consists of an array of :manual:`aggregation stages </changeStreams/#modify-change-stream-output>` as the first
+parameter to filter and transform the change events.
 
 .. code-block:: javascript
 
+   // watch for changes in documents with a `runtime` value less than 15
+
    const pipeline = [ { $match: { runtime: { $lt: 15 } }, ];
    const changeStream = await collection.watch(pipeline);
 
-The ``watch()`` method accepts an additional ``options`` object as the second
-parameter. Refer to the links at the end of this section for more
-information on the settings you can configure in this object.
+The ``watch()`` method accepts an ``options`` object as the second parameter. Refer to the links at the end of this
+section for more information on the settings you can configure with this object.
 
-The ``watch()`` method returns an instance of a ``ChangeStream``. You can
-call methods on the ChangeStream such as ``hasNext()`` to check for remaining
-documents in the stream, ``next()`` to request the next document in the
-stream, ``pause()`` to stop emitting events, ``resume()`` to continue to
-emit events, and ``close()`` to close the ChangeStream. You can also attach
-listener functions by calling the ``on()`` method on the instance. See the
-link to the ``ChangeStream`` API documentation below for more details on the
-available methods.
+The ``watch()`` method returns an instance of a :node-api:`ChangeStream </ChangeStream.html>`. You can consume change
```

Comment:
I would avoid "consume" since it's a new verb not used in the document before and probably not a familiar term for a Jasmin persona audience.